File: legacy/Quera/College/03. Variable & Math Statements/Exercises/3/3.py
a, b, c, d = int(input()), int(input()), int(input()), int(input())
print("Sum : %.6f" % (a + b + c + d))
print("Average : %6f" % ((a + b + c + d) / 4.))
print("Product : %6f" % (a * b * c * d))
x, y = (a + b + abs(a - b)) / 2, (c + d + abs(c - d)) / 2
print("MAX : %6f" % ((x + y + abs(x - y)) / 2.))
x, y = (a + b - abs(a - b)) / 2, (c + d - abs(c - d)) / 2
print("MIN : %6f" % ((x + y - abs(x - y)) / 2.))

File: legacy/Quera/College/03. Variable & Math Statements/Exercises/3/3.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int a, b, c, d;
    cin >> a >> b >> c >> d;
    cout << setprecision(6) << fixed << "Sum : " << 1.0 * (a + b + c + d) << endl;
    cout << setprecision(6) << fixed << "Average : " << (a + b + c + d) / 4. << endl;
    cout << setprecision(6) << fixed << "Product : " << 1.0 * a * b * c * d << endl;
    int x = (a + b + abs(a - b)) / 2;
    int y = (c + d + abs(c - d)) / 2;
    cout << setprecision(6) << fixed << "MAX : " << (x + y + abs(x - y)) / 2. << endl;
    x = (a + b - abs(a - b)) / 2;
    y = (c + d - abs(c - d)) / 2;
    cout << setprecision(6) << fixed << "MIN : " << (x + y - abs(x - y)) / 2. << endl;
    return 0;
}
